### PR TITLE
balances upgrade + daily aggregation with forward fill

### PR DESCRIPTION
**Pull request by 0xRobin**
- rename some of the columns in the balances schema
- addition of daily aggregation with forward fill

### CODE REVIEW

**Comment by 0xRobin** [comment]
example query: https://dune.com/queries/3475840/5841282

File: macros/models/_sector/tokens/balances_daily.sql [hunk 1]
```diff
@@ -1,23 +1,56 @@
...
+            ON  d.day >= b.day
+            and (b.next_update_day is null OR d.day < b.next_update_day) -- perform forward fill
+)
+
+select
+    b.*
+    ,b.balance * p.price as balance_usd
+from(
+    select * from forward_fill
+    where balance > 1
```
Comment (aalan3) [inline macros/models/_sector/tokens/balances_daily.sql:49]: Should this be`>0`?

File: macros/models/_sector/tokens/balances_enrich.sql [hunk 2]
```diff
@@ -1,47 +1,34 @@
...
+from {{balances_raw}} balances
 left join {{ source('tokens', 'erc20') }} erc20_tokens on
-    erc20_tokens.blockchain = '{{ blockchain }}' AND (
+    erc20_tokens.blockchain = balances.blockchain AND (
     CASE
-        WHEN type = 'erc20' THEN erc20_tokens.contract_address = balances.contract_address
+        WHEN token_standard = 'erc20' THEN erc20_tokens.contract_address = balances.token_address
         -- TODO: should not be hardcoded
-        WHEN type = 'native' THEN erc20_tokens.contract_address = 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+        WHEN token_standard = 'native' THEN erc20_tokens.contract_address = 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
```
Comment (aalan3) [inline macros/models/_sector/tokens/balances_enrich.sql:25]: This will not work for Polygon I think, we should pass this in instead.

**Review by aalan3 — COMMENTED** [review]
Looks great! had a couple of comments. Can you also update the description with a summary of the changes and PR name for posterity

File: macros/models/_sector/tokens/balances_enrich.sql (hunk 2 above)
Comment (0xRobin) [reply macros/models/_sector/tokens/balances_enrich.sql:25]: I have only changed the column name here.

Comment (aalan3) [reply macros/models/_sector/tokens/balances_enrich.sql:25]: We use this for polygon https://github.com/duneanalytics/spellbook/blob/main/tokens/models/tokens/polygon/tokens_polygon_erc20.sql#L14

Most other L2s use ETH, we could actually also just pass in the the symbol and the decimal (it seems to be 8 for BNB?)

**Review by aalan3 — APPROVED** [review]

**Pull request merged**